src/docs/product/data-management-settings/server-side-sampling/index.mdx

```diff
@@ -10,43 +10,41 @@ This feature is in beta and is only available if your organization is participat
 
 </Note>
 
-Capturing a single trace involves minimal overhead, but capturing traces for every page load or every API request may add an undesirable load to your system. Sampling your events allows you to better manage the number of events sent to Sentry, so you can tailor your volume to your organization's needs.
+Capturing a single trace involves minimal overhead, but capturing traces for every page load or every API request may add undesirable load to your system. Sampling your events allows you to better manage the number of events sent to Sentry so you can tailor your volume to your organization's needs.
 
 _Server-side sampling_ lets you control which transactions Sentry retains by setting sampling rules and rates, so you see more of the transactions you want to explore further, and less of the ones you don’t. In contrast to [client-side sampling](/platform-redirect/?next=/configuration/sampling/), server-side sampling allows you to manage your sampling configuration centrally and lets you make changes without the need to reconfigure the SDK and redeploy your application.
 
-![The Server-side Sampling page with a list of rules](server-side-sampling.png)
+![The Server-Side Sampling page with a list of rules](server-side-sampling.png)
 
-> [A trace](/product/sentry-basics/tracing/distributed-tracing/) represents the record of an entire operation you want to measure or track, like page load, an instance of a user completing some action in your application, or a cron job in your backend. A trace consists of one or more transactions and the trace can be distributed across multiple services or projects.
+> _[A trace](/product/sentry-basics/tracing/distributed-tracing/) represents the record of an entire operation you want to measure or track, like a page load, an instance of a user completing some action in your application, or a cron job in your backend. A trace consists of one or more transactions and the trace can be distributed across multiple services or projects._
 
-Because every project is different, you can set multiple sample rules with different sample rates per project. For example, you may need more events from high converting pages, critical API endpoints, or need to focus on latency issues from your latest release.
+Because every project is different, you can set multiple sample rules with different sample rates per project. For example, you may need more events from high converting pages, critical API endpoints, or you may need to focus on latency issues from your latest release.
```

Comment:
```suggestion
While server-side sampling allows you to focus in on smaller number of indexed (or retained) events that you can explore more deeply, it also allows you to send a larger number of processed (or ingested) events to Sentry from which performance metrics are derived. With this larger set of processed events, we're better able to provide you with statistically significant insights into the performance of your application. Learn more in our [full Metrics documentation](/product/sentry-basics/metrics/).
 
Because every project is different, you can set multiple sample rules with different sample rates per project. For example, you may need more events from high converting pages, critical API endpoints, or you may need to focus on latency issues from your latest release.
```